packages/devextreme/js/__internal/core/license/trial_panel.ts

```diff
@@ -104,20 +139,58 @@ class DxLicense extends HTMLElement {
     return link;
   }
 
+  private _createButton(): HTMLDivElement {
+    const button = document.createElement('div');
+    button.style.cssText = this._buttonStyles;
+    button.innerHTML = `<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 16 16">
+  <polygon points="13.4 12.7 8.7 8 13.4 3.4 12.6 2.6 8 7.3 3.4 2.6 2.6 3.4 7.3 8 2.6 12.6 3.4 13.4 8 8.7 12.7 13.4 13.4 12.7"/>
+</svg>`;
+    const polygon = button.querySelector('polygon');
+    const svg = button.querySelector('svg');
+    if (svg) {
+      svg.style.verticalAlign = 'baseline';
```

Comment:
Didn't we use `style.cssText` to comply with CSP? I guess direct assignment to `style.styleName` can violate it?

Reply:
Seems that `_createImportantStyles` shoul be used here